### PR TITLE
Add static content through partials

### PR DESCRIPTION
**Pull request by jackfranklin**
The slug changes just merged in https://github.com/alphagov/transaction-wrappers/pull/3 have also broken this - fix incoming.

Having spoken to Jordan yesterday we agreed to try using partials to manage the content of each individual transaction, which has the advantage of us being able to use proper Ruby conditionals & HTML, no more messing around with Yaml.

This is my first pass at it. I extracted the different form questions, of which there are five:
- Choose a document type (eg Nulla Osta, Certificate of no impediment, etc)
- Postage? Yes/No
- Postage options dropdown
- Registration Count (How many births would you like to register?)
- Document Count (How many certificates / documents do you need?)

Then each transaction has its own folder within `views/partials/transactions` that matches the slug, and defines three partials:
- `_done.html.erb` (once payment has gone through)
- `_intro.html.erb` (first screen, contains form & info)
- `_confirm.html.erb` (page confirming total cost prior to going off to Barclays to pay)

For example, here's the intro for a certificate of marriage:

```
<p>Pay for documents you need from the Foreign &amp; Commonwealth Office (FCO) to prove you’re allowed to get married abroad.</p>

<%= render "partials/questions/document_type", :question_text => "Which type of certificate do you need?" %>
[and so on...]
```

### CODE REVIEW

File: app/views/partials/transactions/pay-for-certificates-for-marriage/_confirm.html.erb [hunk 1]
```diff
@@ -0,0 +1 @@
+<p>The cost will be &pound;[PRICE]</p>
```
Comment (JordanHatch) [inline app/views/partials/transactions/pay-for-certificates-for-marriage/_confirm.html.erb]: Is the total cost supposed to be interpolated here?

File: app/views/partials/transactions/pay-to-deposit-marriage-documents/_confirm.html.erb [hunk 2]
```diff
@@ -0,0 +1 @@
+<p>The cost to deposit your certificates is &pound;[PRICE]</p>
```
Comment (JordanHatch) [inline app/views/partials/transactions/pay-to-deposit-marriage-documents/_confirm.html.erb]: The price is missing here too.

File: app/views/partials/transactions/pay-to-register-birth-abroad/_confirm.html.erb [hunk 3]
```diff
@@ -0,0 +1 @@
+<p>The cost for <%= @calculation.item_list %> is &pound;<%= "%0.2f" % @calculation.total_cost %>.</p>
```
Comment (JordanHatch) [inline app/views/partials/transactions/pay-to-register-birth-abroad/_confirm.html.erb]: If you merge master into this branch, the sprintf formatting has been extracted into a `format_money` helper which adheres to our styleguide over when to use decimal places.

**Comment by JordanHatch** [comment]
I like the general gist of this, but it would be good to merge master into this branch before anything else.

I think the prices in the text should be taken from the existing attributes on the transaction instead of being hard-coded, so that when there is a change in the price, we only have to update it in a single place.

File: app/views/partials/transactions/pay-for-certificates-for-marriage/_confirm.html.erb (hunk 1 above)
Comment (JordanHatch) [inline app/views/partials/transactions/pay-for-certificates-for-marriage/_confirm.html.erb]: Could you not use `@calculation.total_cost`?

**Comment by jackfranklin** [comment]
Cool :)

Just merged in master now and doing some updates based on that - then I'll go through your above comments.

**Comment by jackfranklin** [comment]
@JordanHatch 

> I think the prices in the text should be taken from the existing attributes on the transaction instead of being hard-coded, so that when there is a change in the price, we only have to update it in a single place.

Just added those in the latest commit. Still awaiting content bits for a couple of the transactions (legalisation) but it's all a bit tidier now, tests passing, etc.

**Comment by jackfranklin** [comment]
So I think this is done as much as it can be right now - until we have the story around keeping track of user answers (https://www.pivotaltracker.com/story/show/46267389), I don't think this can progress. Propose we merge this in @JordanHatch ?

File: app/views/partials/questions/_postage_fee_yes_no.html.erb [hunk 4]
```diff
@@ -0,0 +1,6 @@
+<p>
+  <%= label_tag "transaction_postage" do %>
+    <%= raw(question_text) %>
```
Comment (JordanHatch) [inline app/views/partials/questions/_postage_fee_yes_no.html.erb]: Should there be a line break here, as per the other fields?

**Pull request merged**